docs/sources/enterprise/settings-updates.md

```diff
@@ -0,0 +1,73 @@
++++
+title = "Settings updates at runtime"
+description = "Settings updates at runtime"
+keywords = ["grafana", "runtime", "settings"]
+weight = 500
++++
+
+# Settings updates at runtime
+
+> **Note:** Available in Grafana Enterprise v8.0+.
+
+Settings updates at runtime allows you to update Grafana settings with no need to restart the instance.
```

Comment:
I just realized that we should mention that the db settings will override settings from other sources and that if db settings are removed they config will revert to whatever is set in the underlying layers of config.

```db < commandline < env vars < file < defaults)```

Reply:
Added, good catch! Thanks! 👌🏻